tests/unit/tests/utils/utils_part.c/utils_part.c

```diff
@@ -0,0 +1,473 @@
+/*
+ * <tested_file_path>src/utils/utils_part.c</tested_file_path>
+ * <tested_function>_how_much_to_evict</tested_function>
+ * <functions_to_leave>
+ *	INSERT HERE LIST OF FUNCTIONS YOU WANT TO LEAVE
+ *	ONE FUNCTION PER LINE
+ * </functions_to_leave>
+ */
+
+#undef static
+
+#undef inline
+
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+#include "print_desc.h"
+
+#include "ocf/ocf.h"
+#include "../ocf_cache_priv.h"
+#include "../ocf_request.h"
+#include "../metadata/metadata.h"
+#include "../engine/cache_engine.h"
+#include "../eviction/ops.h"
+#include "utils_part.h"
+
+#include "utils/utils_part.c/utils_part_generated_wraps.c"
+
+uint32_t __wrap_ocf_part_get_max_size(ocf_cache_t cache, ocf_part_id_t part_id)
+{
+	return mock();
+}
+
+uint32_t __wrap_ocf_engine_repart_count(struct ocf_request *req)
+{
+	return mock();
+}
+
+uint32_t __wrap_ocf_engine_unmapped_count(struct ocf_request *req)
+{
+	return mock();
+}
+
+uint32_t __wrap_ocf_part_get_reserved(ocf_cache_t cache, ocf_part_id_t part_id)
+{
+	return mock();
+}
+
+uint32_t __wrap_ocf_part_get_occupancy(ocf_cache_t cache, ocf_part_id_t part_id)
+{
+	return mock();
+}
+
+ocf_cache_line_t __wrap_ocf_freelist_num_free(ocf_freelist_t freelist)
+{
+	return mock();
+}
+
+static void _how_much_to_evict_test01(void **state)
```

Comment:
Tests look really comprehensive, I really like it. Please consider adding one for req->part_evict flag - might be convenient to add getter/setter functions for easy mocking.